ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandModal.qml

```diff
@@ -127,28 +139,11 @@ ModalPopup {
                 }
             }
 
-            SVGImage {
-                width: 16
-                height: 16
-                visible: warningText.visible
-                source: "../../../../img/warning.svg"
-                anchors.horizontalCenter: parent.horizontalCenter
-                anchors.bottom: warningText.top
-                anchors.bottomMargin: 4
-            }
-
-            StyledText {
-                id: warningText
-                visible: !root.isRequested
-                //% "You need to request the recipient’s address first.\nAssets won’t be sent yet."
-                text: qsTrId("you-need-to-request-the-recipient-s-address-first--nassets-won-t-be-sent-yet-")
-                color: Style.current.danger
-                horizontalAlignment: Text.AlignHCenter
-                wrapMode: Text.WordWrap
-                anchors.right: parent.right
-                anchors.left: parent.left
+            AddressRequestedValidator {
```

Comment:
Come to think of it, couldn’t we pass the address itself, and set isWarn to true when the address is 0x0 and isRequested is true?